app.html

```diff
@@ -1,22 +1,41 @@
-<!doctype html>
+<!DOCTYPE html>
 
 <html lang="en">
-
-<head>
-    <meta charset="utf-8">
+  <head>
+    <meta charset="utf-8" />
     <title>Contacts List</title>
-    <link rel="stylesheet" href="app.css">
-</head>
+    <link rel="stylesheet" href="app.css" />
+  </head>
 
-<body>
-    <div id="app">
-        <h1>Aaron's Contact List</h1>
-        <div id="contact-list-container">
-            Placeholder for contact list container
-            <div class="search-bar-container"></div>
-        </div>
-        <script src="js/scripts.js"></script>
-    </div>
-</body>
+  <body>
+    <div id="app" class="float-container">
+      <h1>Aaron's Contact List</h1>
+      <div id="contact-list-container" class="float-child">
+        Placeholder for contact list container
+        <div class="search-bar-container"></div>
+        <button type="button" onclick="showNewContactForm()">No Name</button>
+      </div>
 
-</html>
\ No newline at end of file
+      <div id="new-contact-container" class="float-child">
```

Comment:
why are we calling the id "new-contact-container", this should be reused with all contacts not just new